addons/agent_mirrorpeer_controller_test.go

```diff
@@ -253,6 +253,16 @@ func GetTestCephBlockPool() *rookv1.CephBlockPool {
 	}
 }
 
+func GetOppositePeerRefs(mp *multiclusterv1alpha1.MirrorPeer, spokeClusterName string) []multiclusterv1alpha1.PeerRef {
+	peerRefs := make([]multiclusterv1alpha1.PeerRef, 0)
+	for _, v := range mp.Spec.Items {
+		if v.ClusterName != spokeClusterName {
+			peerRefs = append(peerRefs, v)
+		}
+	}
+	return peerRefs
+}
+
```

Comment:
Why is this removed from the controller file and moved to test ?

Reply:
Because it is being used by the test in this file only.